lib/chiketto.rb

```diff
@@ -13,6 +13,8 @@ module Chiketto
   require 'chiketto/attendee_profile'
   require 'chiketto/category'
   require 'chiketto/email'
+  require 'chiketto/choice'
+  require 'chiketto/question'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.